otj-metrics-actuate-15/pom.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
4. Discovery -> CompatibilitySHim
	- compatibilityshhim also includes sprint bopot 1.5 metrics
	- move the java.net to avro module
	ServiceInfo, announcementregistry - leave as is but probably move to layer
	
5. Archive metrics-spring
6. Metrics move most of it to otj-metrics
	Keep micrometer in your pom for now, we'll ponder
	drop servicestarted or compatibilityshim